__tests__/StarkwareLib.test.ts

```diff
@@ -1,9 +1,10 @@
+import Web3 from 'web3';
 import { provider as Provider } from 'web3-core';
 
+import usdcAbi from '../src/contracts/usdc-abi.json';
 import { StarkwareLib } from '../src/index';
-import { DummyProvider } from './helpers/DummyProvider';
 import { Networks } from '../src/types';
-import Web3 from 'web3';
+import { DummyProvider } from './helpers/DummyProvider';
 
 // async ropsten calls can take >10k MS
```

Comment:
async goerli

Reply:
can remove this since nsb default is 30s

Reply:
done